src/css/largeCSS.scss

```diff
@@ -0,0 +1,52 @@
+// Media Query for screen size over 1500px;
```

Comment:
I'm looking forward to working with sass a little more - interesting to see the separate files... Seems like a much better way to organize it all. Definitely an improvement over the super long css files I'm used to.